daml-lf/spec/value.rst

```diff
@@ -560,14 +552,19 @@ Both ``key`` and ``value`` are required.
 message GenMap
 ^^^^^^^^^^^
 
-*since version 11*
+(*since version 11*)
 
 A map where keys and values are homogeneous.
 
 In this version, these fields are included:
 
 * repeated `message GenMap.Entry`_ entries
 
-The ``keys`` and the ``value`` fields of every member of ``entries`` must
-conform to the same types. Entries occur in insertion order. If two ore more
-entries have the same keys, the last one override the former entries.
+.. note: *this section is non-normative*
+
+   The ``value`` field of every member of ``entries`` must conform to
```

Comment:
it seems like you deleted some of those type of notes completely while others got moved into a non-normative section. Is there any specific reason why you chose one or the other in the various places?

Reply:
I put back the few I deleted